hack/run-in-python-container.sh

```diff
@@ -28,6 +28,6 @@ docker run \
     --rm -i \
     -e HOME=/tmp \
     -e PYTHONPATH='' \
-    -v "${REPO_ROOT:?}:${REPO_ROOT:?}" -w "${REPO_ROOT}" \
+    -v "${REPO_ROOT:?}:${REPO_ROOT:?}:Z" -w "${REPO_ROOT}" \
```

Comment:
I don't fully understand what this does, but the docs warn that this could be dangerous: https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label
/cc @BenTheElder

Reply:
Ok, I'll take that approach. A few notes though

- Unlike in #5983, we would only be relabelling the source repo here, which I guess is unlikely to cause problems - maybe we would want to make a temporary copy of the repo if we really wanted to relabel though
- It looks like `--security-opt="label=disable"` is the correct syntax according to both docker and podman docs
- See [mkproj](https://github.com/kubernetes/test-infra/pull/16241#discussion_r378534883) for an example of where we did use the relabelling approach